.travis.yml

```diff
@@ -24,7 +24,7 @@ before_script:
 
   # Install composer dependencies
   - composer validate
-  - composer require silverstripe/recipe-cms 1.0.x-dev  --no-update
+  - composer require silverstripe/recipe-cms 1.1.x-dev  --no-update
```

Comment:
Is this something we should introduce a `$RECIPE_VERSION` var to?
This is already stable so support will need to continue for the lower versions too

Reply:
See #215 (noted requirement)